ClientApp/src/components/Pagination.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const Pagination = ({ listingsPerPage, totalListings, paginate }) => {
+    const pageNumbers = [];
+
+    for (let i = 1; i <= Math.ceil(totalListings / listingsPerPage); ++i) {
+        pageNumbers.push(i);
+    }
+    return (
+        <nav>
+            <ul className="pagination">
+                {pageNumbers.map(number => (
+                    <li key={number} className="page-item">
+                        <a onClick={() => paginate(number)} className="page-link">
```

Comment:
Suggestion: change 'a' tag to 'button', so that the cursor will display the click icon.

Reply:
On what browser does the 'a' tag not show you the click icon? It shows it fine for me(changes to click cursor when hovering over) on Chrome & Firefox.